blockware/matrix/lib/MatrixRain/MatrixRain.cpp

```diff
@@ -0,0 +1,225 @@
+#include <Adafruit_SSD1351.h>
```

Comment:
Did you write this or was it forked from somewhere else? If forked, let's include a link aback to the original.

Reply:
the MatrixRain? it’s all from me I just turned the visual into code